jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/spec/client/instance/JAXRSClientIT.java

```diff
@@ -71,7 +70,6 @@ public JAXRSClientIT() {
 
   }
 
-
```

Comment:
It's not an Arquillian test so it would produce an assertion failure as the URL would be `null`. I don't see the ports/host being used here so it's likely not a big issue here.